include/deal.II/distributed/fully_distributed_tria_util.h

```diff
@@ -64,6 +64,41 @@ namespace parallel
         const bool         construct_multilevel_hierarchy = false,
         const unsigned int my_rank_in = numbers::invalid_unsigned_int);
 
+
+      /**
+       * Construct a parallel::fullydistributed::ConstructionData. In contrast
```

Comment:
If you drop this part from the pull request again, then I'll approve it :-)

Reply:
Sorry about this ;)